OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/Topic.java

```diff
@@ -0,0 +1,33 @@
+package com.optimove.android.optimobile;
+
+import java.util.List;
+
+public class Topic {
```

Comment:
all classes in android sit in 1 folder (think because how access modifiers work in android, would have to make everything public if diff folders)

so, mb can call this PreferenceCenterTopic to group them all together and have some context